packages/carbon-graphs/src/js/carbon.js

```diff
@@ -41,6 +42,7 @@ const tools = {
  */
 const helpers = {
   AXIS_TYPE,
+  BUBBLE,
```

Comment:
May I know why do we need this bubble in carbon.js file?

Reply:
So we can access the bubble palette contants:
<img width="361" alt="CleanShot 2021-05-05 at 17 53 54@2x" src="https://user-images.githubusercontent.com/38024451/117219444-e6ac9080-adca-11eb-8d75-7dd652c028a3.png">